WordPress/Classes/Utility/AccountHelper.swift

```diff
@@ -43,14 +42,13 @@ import Foundation
     }
 
     static func logBlogsAndAccounts(context: NSManagedObjectContext) {
-        let accountService = AccountService(managedObjectContext: context)
         let blogService = BlogService(managedObjectContext: context)
         let allBlogs = blogService.blogsForAllAccounts()
         let blogsByAccount = Dictionary(grouping: allBlogs, by: { $0.account })
 
-        let defaultAccount = accountService.defaultWordPressComAccount()
+        let defaultAccount = try? WPAccount.lookupDefaultWordPressComAccount(in: context)
 
-        let accountCount = accountService.numberOfAccounts()
+        let accountCount = (try? WPAccount.lookupNumberOfAccounts(in: context)) ?? 0
```

Comment:
Good point! [The one for Objective-C](https://github.com/wordpress-mobile/WordPress-iOS/blob/93d5bf4cf8f1853cae63a859ba0bf7159fab0070/WordPress/Classes/Models/WPAccount%2BLookup.swift#L121) does exactly this. I'm inclined to make this change, what do you think @jkmassel?

Reply:
So IMHO this is a symptom of poor error handling throughout the app, but is also a step toward the right direction though it feels weird because it's half-done.

My thought – anything that does IO (or depends on something that does IO, but isn't UI) should `throw`. That's so that we don't do things like "return 0 if there's an error" – that swallows the error, so a user might wind up with an empty list instead of an error message like "Network Error – Unable to Connect" because the error was thrown away before it reached the UI layer. 

The reason the `try? [...] ?? [default]` pattern is here is because the parent method isn't marked `throws` – this can be for a few reasons, including Obj-C interop, and a desire to keep small changesets.

IMHO though, this is a step in the right direction, because a search for `try?` in the app will reveal cases where we need to refactor for error handling. 

Anyway, happy to hear y'alls thoughts!